src/packet/tcp.rs.in

```diff
@@ -10,11 +10,40 @@ use packet::Packet;
 use packet::PacketSize;
 use packet::PrimitiveValues;
 use packet::ip::IpNextHeaderProtocol;
+use ::util::Octets;
 
 use pnet_macros_support::types::*;
 
 use std::net::{Ipv4Addr, Ipv6Addr};
 
+/// NS – ECN-nonce concealment protection (experimental: see RFC 3540).
+pub const FLAG_NS: u9be  = 0b100000000;
```

Comment:
Could you expand the names too? I'd rather have more verbose code with readable constants, than save a few characters.